src/sage/algebras/lie_algebras/lie_algebra.py

```diff
@@ -616,28 +616,47 @@ def __getitem__(self, x):
 
     def _coerce_map_from_(self, R):
         """
-        Return ``True`` if there is a coercion from ``R`` into ``self`` and
-        ``False`` otherwise.
+        Return a coercion map from ``R`` into ``self`` if one exists,
+        or ``True`` if a generic coercion exists, or ``False`` otherwise.
 
         The things that coerce into ``self`` are:
 
         - Lie algebras in the same variables over a base with a coercion
           map into ``self.base_ring()``.
 
         - A module which coerces into the base vector space of ``self``.
+          In this case, the coercion map is given by :meth:`from_vector`
+          (composed with the appropriate coercion on the base module).
 
         TESTS::
 
             sage: L.<x,y> = LieAlgebra(QQ, abelian=True)
             sage: L._coerce_map_from_(L.module())
+            <bound method ...from_vector of Abelian Lie algebra on 2 generators (x, y) over Rational Field>
+            sage: cm = L._coerce_map_from_(FreeModule(ZZ, 2))
```

Comment:
```suggestion
            sage: L.has_coerce_map_from(L.module())
            True
            sage: cm = L.coerce_map_from(FreeModule(ZZ, 2))
```
These are better tests IMO (despite being indirect).